index.js

```diff
@@ -105,7 +160,18 @@ function findById() {}
  *  filterByGenre(movies, "Horror")
  *  //> []
  */
-function filterByGenre() {}
+function filterByGenre(movies, genre) {
+  let movie = [];
```

Comment:
Consider a plural variable like `movies` here since we are expecting numerous values